### DISCUSSION TITLE
Android使用PagImageView播放bmp合成的pag部分手机出现播不出来和不动的情况，整个项目的pag都没有播成功

### DISCUSSION THREAD

**Discussion by MrHayden** [Tencent/libpag#2084] [Q&A]
![image](https://github.com/Tencent/libpag/assets/19620807/e669f989-90cd-40e8-b1a4-c0ba87a5425e)
pag版本 4.3.45
手机信息：
![image](https://github.com/Tencent/libpag/assets/19620807/d5875a6a-040f-46dc-874d-47d561e68707)
，无报错信息

**Comment by MrHayden** [comment]
资源：
[Uploading skill_rank_big_animation_normal_10.zip…]()

**Comment by kevingpqi123** [comment]
这里问题建议先基于 pag 的 官方 demo做测试，如果官方 demo 是 OK 的，对比实现有哪些差异

**Reply by MrHayden** [reply]
官方demo测试的也是不行

**Reply by MrHayden** [reply]
是因为动效是bmp合成导致的问题么

**Comment by MrHayden** [comment]
https://github.com/Tencent/libpag/assets/19620807/cfe39544-5868-4f02-9786-5ba9d1a6c641

这个是demo的测试视频，这个上面的截图手机所有的pag都播放不了

**Comment by MrHayden** [comment]
通过云测测试这个手机都挺正常的，但是无法匹配到这个用户手机的版本号的手机
![image](https://github.com/Tencent/libpag/assets/19620807/f09f53b3-8e6f-45f4-8b7c-1e4166189ef5)
差异只有这个，最后这个用户的手机通过还原手机的所有设置后能正常播放了，不知道用户之前是设置了什么
